lib/measures/tbd/resources/psi.rb

```diff
@@ -149,6 +149,173 @@ def append(p)
   end
 end
 
+def processTBDinputs(surfaces, edges, set, io_f, schema_f)
```

Comment:
Is `io_f` an `IO` object or a file path?  It looks like a file path?  If so I'd suggest naming it path or something

Reply:
Can do.